client/fingerprint/cpu.go

```diff
@@ -33,15 +33,15 @@ func (f *CPUFingerprint) Fingerprint(cfg *config.Config, node *structs.Node) (bo
 	}
 
 	mhz := stats.CPUMHzPerCore()
-	node.Attributes["cpu.frequency"] = fmt.Sprintf("%.6f", mhz)
-	f.logger.Printf("[DEBUG] fingerprint.cpu: frequency: %02.1f MHz", mhz)
+	node.Attributes["cpu.frequency"] = fmt.Sprintf("%.1f", mhz)
```

Comment:
Since the values are floored, we will loose the precision so `%.1f` would not any value.